src/cswinrt/code_writers.h

```diff
@@ -2611,6 +2647,22 @@ private static % _% = new %("%.%", %.IID);
     void write_activation_factory_objref_definition(writer& w, TypeDef const& classType)
     {
         auto objrefname = w.write_temp("%", bind<write_objref_type_name>(classType));
+        if (settings.reference_projection)
+        {
+            w.write(R"(
+private static WindowsRuntimeObjectReference %
```

Comment:
`WindowsRuntimeObjectReference` is a private implementation detail type, so we shouldn't use it in the ref assembly. Can we skip emitting this property entirely, since it's private anyway? Like, we shouldn't need to emit any non-public members in general.